src/components/FilterPanel/FilterNameField.tsx

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import { TextField } from '@mui/material';
 import { useTheme } from '@mui/material/styles';
 import { Controller } from 'react-hook-form';
+import { CharacterFilterControl } from '../../types/types';
```

Comment:
[nth] it seems that `types` could be extracted to the uppper level and `types` folder could be removed